server/ast/column_table_def.go

```diff
@@ -133,5 +131,24 @@ func nodeColumnTableDef(node *tree.ColumnTableDef) (_ *vitess.ColumnDefinition,
 			GeneratedExpr: generated,
 			Stored:        generatedStored,
 		},
-	}, nil
+	}
+	var checkConstraints = make([]*vitess.ConstraintDefinition, len(node.CheckExprs))
+	if len(node.CheckExprs) > 0 {
+		for i, checkExpr := range node.CheckExprs {
+			expr, err := nodeExpr(checkExpr.Expr)
+			if err != nil {
+				return nil, nil, err
+			}
+			checkConstraints[i] = &vitess.ConstraintDefinition{
+				Name: string(checkExpr.ConstraintName),
+				Details: &vitess.CheckConstraintDefinition{
+					Expr:     expr,
+					Enforced: true,
+				},
+			}
+		}
+		// TODO: vitess does not support multiple check constraint on a single column
```

Comment:
Instead of silently dropping the other check constraints, I think it's better to fail fast here and return an error if the user specified multiple check constraints and we can't support all of them. That way, the gap is more visible and if customers hit it, they get a clear error message and can ask us to fix it, instead of their SQL looking like it succeeded but not working as they expect it to.